src/cloudai/systems/slurm/strategy/slurm_command_gen_strategy.py

```diff
@@ -205,8 +229,11 @@ def _write_sbatch_script(self, args: Dict[str, Any], env_vars_str: str, srun_com
             batch_script_content.append(f"#SBATCH --account={args['account']}")
         if "distribution" in args:
             batch_script_content.append(f"#SBATCH --distribution={args['distribution']}")
-        if "gpus_per_node" in args:
+
+        gpus_per_node = args.get("gpus_per_node")
+        if gpus_per_node:
             batch_script_content.append(f"#SBATCH --gpus-per-node={args['gpus_per_node']}")
+            batch_script_content.append(f"#SBATCH --gres=gpu:{args['gpus_per_node']}")
```

Comment:
_[minor]_ Using `gpus_per_node` instead of `args["gpus_per_node"]` will improve redability. (same for the line above)

Reply:
done